### PR TITLE
MineClone2 support

### PR DESCRIPTION
**Pull request by tacotexmex**
This PR provides MCl2 support by offering MCL2-compatible craft recipes.

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -86,39 +88,63 @@ minetest.register_tool("orbs_of_time:orb_dusk",{
...
+	  },
+	})
+
+	minetest.register_craft({
+	  output = "orbs_of_time:orb_dusk 2",
+	  recipe = {
+	   {"orbs_of_time:orb_night"},
+	   {"orbs_of_time:orb_day"},
+	  },
+	})
```
Comment (SmallJoker) [inline init.lua]: Could you please fix these indents? Same for L113-L129

File: init.lua [hunk 2]
```diff
@@ -32,6 +33,7 @@ minetest.register_tool("orbs_of_time:orb_night",{
 	inventory_image = "orbs_orb_night.png",
 	wield_image = "orbs_orb_night_weild.png",
 	stack_max=1,
+	groups = { tool=1 },
```
Comment (SmallJoker) [inline init.lua:36]: Why is this group required? `register_tool` already makes it obvious that it's a tool.

**Review by SmallJoker — COMMENTED** [review]
Is it still up to date with the MCl2 changes? :+1: after the comments are handled.

File: init.lua (hunk 2 above)
Comment (tacotexmex) [reply init.lua:36]: Just following MCL2 group naming convention, I believe.

**Pull request merged**